main.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <string>
+#include <vector>
+using namespace std;
+
+void print(vector<string> & sent){
+    for(int i = 0; i < sent.size() - 1 ; i++)
+    {
+        cout << sent[i] << " ";
+    }
+    cout << sent[sent.size()-1];
+}
+
+bool check(vector<string>& wordChecks){
+    int index = 0;
+    if(wordChecks[index] != "the" && wordChecks[index] != "The"){
+        cout << "ERROR 1! The sentence '";
+        print(wordChecks);
+        cout << "' does not meet requirements" << endl;
+        return false;
+    }
+
+    index++;
+    if(wordChecks[index] != "lazy" && wordChecks[index] != "smelly")
```

Comment:
why not just move the while loop up to here. If there are no adjectives, it will fall through, otherwise it will consume them...